client/src/app/site/deployment-center/deployment-center-setup/wizard-logic/deployment-center-state-manager.ts

```diff
@@ -61,15 +62,16 @@ export class DeploymentCenterStateManager implements OnDestroy {
   public isFunctionApp = false;
   public vstsKuduOnly = false;
   public vsoAccounts: VSOAccount[] = [];
+
   constructor(
     private _cacheService: CacheService,
     private _azureDevOpsService: AzureDevOpsService,
     private _translateService: TranslateService,
+    private _localStorageService: LocalStorageService,
```

Comment:
Is this backed by session storage or local storage? From what I read, session storage might be a better fit for our scenario.

Reply:
I think it is backed by local storage. there is no session storage service in the extension code. Session is better fit, and that's why to work around I am using sessionId to detech whether to use the stored permission value or to fetch it again.